src/sketch.h

```diff
@@ -790,9 +790,9 @@ class Constraint : public ConstraintBase {
     static hConstraint TryConstrain(Constraint::Type type, hEntity ptA, hEntity ptB,
                                     hEntity entityA, hEntity entityB = Entity::NO_ENTITY,
                                     bool other = false, bool other2 = false);
-    static void ConstrainArcLineTangent(Constraint *c, Entity *line, Entity *arc);
-    static void ConstrainCubicLineTangent(Constraint *c, Entity *line, Entity *cubic);
-    static void ConstrainCurveCurveTangent(Constraint *c, Entity *eA, Entity *eB);
+    static bool ConstrainArcLineTangent(Constraint *c, Entity *line, Entity *arc);
+    static bool ConstrainCubicLineTangent(Constraint *c, Entity *line, Entity *cubic);
+    static bool ConstrainCurveCurveTangent(Constraint *c, Entity *eA, Entity *eB);
 };
```

Comment:
Is this just whitespace in sketch.h? Removing tabs?

Reply:
It's changing the return type (void -> bool)

Reply:
As @vespakoen explained ;-)

Reply:
@ruevs  my dumb.  Not sure how I missed that.